.circleci/check-testnet-deployment.js

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env node
+
+const { waitUntilSync, getBlockHeight, getCommit } = require('@orbs-network/orbs-nebula/lib/metrics');
+const fetch = require('node-fetch');
+
+const TOPOLOGY = 'https://s3.us-west-2.amazonaws.com/boyar-testnet-bootstrap/boyar/config.json';
+const TARGET_HASH = process.argv[2];
+
+if (!TARGET_HASH) {
+    console.log('No target hash to check');
+    process.exit(1);
+}
+
+async function getTopology() {
+    const result = await fetch(TOPOLOGY);
+    const topology = await result.json();
+    return topology;
+}
+
+async function eventuallyDeployed({ chainId, nodes }) {
+    // First let's poll the nodes for the correct version
+    // Until we meet the correct version on all nodes or we bail by the timeout defined
+    let versionDeployed = false;
+    let runsCount = 0;
+    const maxRetries = 10; // 5 minutes
+
+    do {
+        if (runsCount + 1 > maxRetries) {
+            console.log('Max retries limit reached for chainId: ', chainId);
+            break;
+        }
+
+        runsCount++;
+        console.log('Polling all nodes for their version on chainId ', chainId);
+        const results = await Promise.all(nodes.map(({ ip }) => {
+            return getCommit(`${ip}/vchains/${chainId}`);
+        }))
+            .catch(err => {
+                console.log('failed getting the commit hash of one of the nodes for some reason');
+                console.log('error provided:', err);
+            });
+
+        console.log('Got: ', results);
+        if (results.filter(version => version === TARGET_HASH).length === nodes.length) {
+            // Version updated on all nodes
+            versionDeployed = true;
+            continue;
+        }
+
+        console.log('Sleeping for 30 seconds..');
+        await new Promise((r) => { setTimeout(r, 30 * 1000); });
+    } while (versionDeployed === false);
+
+    return {
+        ok: versionDeployed
+    };
+}
+
+async function eventuallyClosingBlocks({ chainId, nodes }) {
+    const firstEndpoint = `${nodes[0].ip}/vchains/${chainId}`;
+
+    // First let's get the current blockheight and wait for it to close another 50 blocks.
+    const currentBlockheight = await getBlockHeight(firstEndpoint);
+
+    try {
+        await waitUntilSync(firstEndpoint, currentBlockheight + 50);
```

Comment:
50 is too long, blocks are closed at a rate of 9 seconds on an idle system. It would be 7 minutes after we waited for the network to start producing blocks, which itself can take 10-15 minutes.

Reply:
I'm updating it to `10`